extensions/sql-database-projects/src/common/promise.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Deferred promise
+ */
+export class Deferred<T> {
```

Comment:
you haven't used this anywhere and probably won't need it

Reply:
Yup; removed references, but forgot to delete the file itself.